### PR TITLE
Added AWS and Digital Ocean droplet issue

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added AWS and Digital Ocean droplet issue

### CODE REVIEW

File: docs/Reference/CLI/CLI-Syntax.md [hunk 1]
```diff
@@ -2131,14 +2131,25 @@ sync-mode="FAST"
 The synchronization mode. The options are `FAST` and `FULL`. The default is `FULL`.
 
 !!! note
-
-    If fast synchronization fails, usually because the node can't find enough valid peers, fast
-    synchronization restarts after a short delay.
+    
+    On AWS, a known [RocksDB](https://github.com/facebook/rocksdb/issues/6435) issue causes fast sync
```
Comment (timbeiko) [inline docs/Reference/CLI/CLI-Syntax.md:2135]: Should we change "On AWS" for "When running Besu on certain cloud providers, notably AWS and Digital Ocean,"

File: docs/Reference/CLI/CLI-Syntax.md [hunk 2]
```diff
@@ -2131,14 +2131,25 @@ sync-mode="FAST"
...
+    to fail occassionally. The following error is displayed repeatedly: 
+    
+    ```
+    ...
+    EthScheduler-Services-1 (importBlock) | ERROR | PipelineChainDownloader | Chain download failed. Restarting after short delay.
+    java.util.concurrent.CompletionException: org.hyperledger.besu.plugin.services.exception.StorageException: org.rocksdb.RocksDBException: block checksum mismatch:
+    ....
+    ```
+    
+    A full restart of the AWS VM is required to restart the fast sync.
```
Comment (timbeiko) [inline docs/Reference/CLI/CLI-Syntax.md:2145]: Again, here I would rephrase to say that a full restart is the way to go for AWS and we have no known solution for Digital Ocean, but are investigating some options, as per https://github.com/hyperledger/besu/issues/591#issuecomment-610015508

File: docs/Reference/CLI/CLI-Syntax.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Reference/CLI/CLI-Syntax.md:2135]: Updated

File: docs/Reference/CLI/CLI-Syntax.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Reference/CLI/CLI-Syntax.md:2145]: Updated
